Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM ubuntu:14.04
+FROM progrium/busybox
```

Comment:
So I have conflicted feelings about trusting user built images.  Not to say anyone would do something malicious but since we're trusting a user image along with a tag that could be overwritten (latest) might open it up for bad things to happen.